core/toolbox.js

```diff
@@ -273,7 +296,10 @@ Blockly.Toolbox.prototype.getClientRect = function() {
   var x = toolboxRect.left;
   var y = toolboxRect.top;
   var width = this.getWidth();
-  var height = toolboxRect.height;
+  // The height here is the combined height of the category menu and flyout.
```

Comment:
This should not include the height of the flyout.  This is the deletion rect for the toolbox.  The flyout is responsible for its own deletion rect.